apps/web/src/services/asset-inventory/components/MetricExplorerChart.vue

```diff
@@ -92,13 +91,9 @@ const setChartData = debounce(async () => {
 
     state.legends = getLegends(rawData, _groupBy);
     if (state.selectedChartType === CHART_TYPE.LINE) {
-        state.chartData = getXYChartData(rawData, _granularity, _period, _groupBy);
-    } else if (state.selectedChartType === CHART_TYPE.COLUMN) {
-        // TODO: implement column chart
-    } else if (state.selectedChartType === CHART_TYPE.DONUT) {
-        state.chartData = getDonutChartData(rawData, _groupBy);
-    } else if (state.selectedChartType === CHART_TYPE.TREEMAP) {
-        state.chartData = getRefinedTreemapChartData(rawData, _groupBy);
+        state.chartData = getRefinedMetricXYChartData(rawData, _granularity, _period, _groupBy);
+    } else {
+        state.chartData = getRefinedMetricRealtimeChartData(rawData, _groupBy);
```

Comment:
description: donut, treemap, column -> realtime chart, so I unified type and helper!!